### PR TITLE
Added more content to System_Design

### PR DESCRIPTION
**Pull request by surbhitawasthi**

### CODE REVIEW

File: System_Design/README.MD [hunk 1]
```diff
@@ -10,4 +10,11 @@ System design is the process of designing the elements of a system such as the a
 * Caching
 * Redundancy and Replication
 
+### Designing a System? Remember these
+* Networking both within processes and among different systems.
```
Comment (Abhishek1103) [inline System_Design/README.MD]: You can rephrase this point as:
* Communication among both process and different systems/services.

> You may/may not ignore the above suggestions :wink:

**Review by Abhishek1103 — COMMENTED** [review]
Some suggestions to rephrase some of the points.

File: System_Design/README.MD (hunk 1 above)
Comment (surbhitawasthi) [reply System_Design/README.MD]: Sure thing!

Comment (surbhitawasthi) [reply System_Design/README.MD]: Done, merging now

**Pull request merged**